### PR TITLE
[Improvement-8295][Document] Standalone Deployment doc should instruct users to configure sudo without password

### LINKED ISSUE #8295: [Improvement][Document] Standalone Deployment doc should instruct users to configure sudo without password
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* In the `Installation` session of ds doc, `Standalone Deployment` currently does not instruct users to configure `sudo` without password while `Pseudo-Cluster Deployment` does, which may cause confusion and is not user-friendly.
* As a new user to ds, after reading the installation doc, I thought in `standalone` mode, I didn't need to configure `sudo` without password. As a result of this misunderstanding, I failed to run shell tasks and spent some time debugging. There could be  error log like below:

```
org.apache.dolphinscheduler.common.shell.AbstractShell$ExitCodeException: sudo: a terminal is required to read the password; either use the -S option to read from standard input or configure an askpass helper

	at org.apache.dolphinscheduler.common.shell.AbstractShell.runCommand(AbstractShell.java:207)
	at org.apache.dolphinscheduler.common.shell.AbstractShell.run(AbstractShell.java:122)
	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execute(ShellExecutor.java:126)
	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execCommand(ShellExecutor.java:103)
	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execCommand(ShellExecutor.java:86)
	at org.apache.dolphinscheduler.common.utils.OSUtils.exeShell(OSUtils.java:400)
	at org.apache.dolphinscheduler.common.utils.OSUtils.exeCmd(OSUtils.java:389)
	at org.apache.dolphinscheduler.common.utils.OSUtils.createMacUser(OSUtils.java:308)
	at org.apache.dolphinscheduler.common.utils.OSUtils.createUser(OSUtils.java:268)
	at org.apache.dolphinscheduler.common.utils.OSUtils.createUserIfAbsent(OSUtils.java:248)
	at org.apache.dolphinscheduler.server.worker.processor.TaskExecuteProcessor.process(TaskExecuteProcessor.java:143)
	at org.apache.dolphinscheduler.remote.handler.NettyServerHandler.lambda$processReceived$0(NettyServerHandler.java:129)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

fix #8295

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by Gallardot — DISMISSED** [review]

File: docs/docs/en/guide/installation/standalone.md [hunk 1]
```diff
@@ -14,6 +14,31 @@ If you want to deploy DolphinScheduler in production, we recommend you follow [c
...
+
+# Modify directory permissions and grant permissions for user you created above
+chown -R dolphinscheduler:dolphinscheduler apache-dolphinscheduler-*-bin
+chmod -R 755 apache-dolphinscheduler-*-bin
+```
+
+> **_NOTICE:_**
+>
+> - Due to DolphinScheduler's multi-tenant task switch user using command `sudo -u {linux-user} -i`, the deployment user needs to have `sudo` privileges and be password-free. If novice learners don’t understand, you can ignore this point for now.
+> - If you find the line "Defaults requirett" in the `/etc/sudoers` file, please comment the content.
```
Comment (Radeity) [inline docs/docs/en/guide/installation/standalone.md]: requirett  -> requiretty

File: docs/docs/zh/guide/installation/standalone.md [hunk 2]
```diff
@@ -12,6 +12,31 @@ Standalone 仅适用于 DolphinScheduler 的快速体验.
...
+
+# 修改目录权限，使得部署用户对二进制包解压后的 apache-dolphinscheduler-*-bin 目录有操作权限
+chown -R dolphinscheduler:dolphinscheduler apache-dolphinscheduler-*-bin
+chmod -R 755 apache-dolphinscheduler-*-bin
+```
+
+> **_注意:_**
+>
+> - 因为任务执行服务是以 `sudo -u {linux-user} -i` 切换不同 linux 用户的方式来实现多租户运行作业，所以部署用户需要有 sudo 权限，而且是免密的。初学习者不理解的话，完全可以暂时忽略这一点
+> - 如果发现 `/etc/sudoers` 文件中有 "Defaults requirett" 这行，也请注释掉
```
Comment (Radeity) [inline docs/docs/zh/guide/installation/standalone.md]: ditto

**Review by Radeity — CHANGES_REQUESTED** [review]

File: docs/docs/en/guide/installation/standalone.md (hunk 1 above)
Comment (SbloodyS) [reply docs/docs/en/guide/installation/standalone.md]: Fixed.

File: docs/docs/zh/guide/installation/standalone.md (hunk 2 above)
Comment (SbloodyS) [reply docs/docs/zh/guide/installation/standalone.md]: Fixed.

**Review by Radeity — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]

**Pull request merged**